packages/imperative/CHANGELOG.md

```diff
@@ -389,6 +389,15 @@ All notable changes to the Imperative package will be documented in this file.
 
 - Major: First major version bump for V3
 
+## `5.25.0`
+
+- Enhancement: Added `ProfileInfo.profileManagerWillLoad` function to verify the credential manager can load. [#2111](https://github.com/zowe/zowe-cli/issues/2111)
```

Comment:
Maybe for straightforward port-PRs like this one, could/should we at least have a pointer to the specific version where the enhancement/fixes where made?
For example:

```
## Recent Changes

- Update: See `5.25.0` for details
```